tests/general/LineContinuationBearTest.py

```diff
@@ -0,0 +1,43 @@
+import logging
+from queue import Queue
+
+from bears.general.LineContinuationBear import LineContinuationBear
+from coalib.testing.LocalBearTestHelper import LocalBearTestHelper
+from coalib.testing.LocalBearTestHelper import execute_bear
+from coalib.results.Result import Result
+from coalib.settings.Section import Section
+from coalib.settings.Setting import Setting
+
+
+good_file = """
+a = some_function(
+    '1' + '2')
+"""
+
+
+class LineContinuationBearTest(LocalBearTestHelper):
+
+    def setUp(self):
+        self.section = Section('name')
+        self.uut = LineContinuationBear(self.section, Queue())
+
+    def test_bear_run(self):
```

Comment:
split into test_good_file and test_bad_file , so when these fail we have a better idea of what has happened